### DISCUSSION TITLE
Question on how you retrieve the queued workflows on the org level

### DISCUSSION THREAD

**Discussion by rajbos** [actions/actions-runner-controller#1050] [General]
Hi, I'm trying to find out how you are retrieving the queued workflows on the organization level through the API's? I cannot find it anywhere, but see references to the GitHubClient in the code. What API are you using for this?

**Comment by toast-gear** [comment]
We aren't, there isn't a means for doing so currently

**Reply by rajbos** [reply]
Ah, I was thinking of how the runners scale then (without the webhook). 

But the scaling happens on looking at the runners themselves of course 🙈.
